spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -170,18 +171,30 @@ def __init__(self, parent=None, background_color=None):
         self.thumbnails_sb.sig_redirect_stdio_requested.connect(
             self.sig_redirect_stdio_requested)
 
+        # Widget empty pane
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "plots",
+            _("No plots generated yet"),
```

Comment:
```suggestion
            _("No plots to show"),
```

There's lots of reasons this pane can become empty after plots have in fact been generated—deleting existing plots (as I've seen many users do frequently), switching consoles, restarting consoles, opening a new one, etc. 

"No plots to show" accurately and concisely conveys there are no figures to display currently, while not making incorrect and potentially confusing presumptions about why this is the case. It also follows the common convention I've seen elsewhere to describe this case in various UIs for empty views, "No X to show" ("No results to show", "No images to show, etc, which can also be re-used for other panes where appropriate).

Additionally, this also better covers the case of the user generating plots but the Plots pane not supporting them, while the description below still makes clear how users _can_ generate more plots to show.